packages/context-entries/lib/Keys.ts

```diff
@@ -64,6 +65,17 @@ export const KeysHttpProxy = {
   httpProxyHandler: new ActionContextKey<IProxyHandler>('@comunica/actor-http-proxy:httpProxyHandler'),
 };
 
+export const KeysHttpCache = {
+  /**
+   * Will skip the HTTP cache if this is set to true
+   */
+  doNotCheckHttpCache: new ActionContextKey<boolean>('@communica/actor-http-cache:do-not-check-http-cache'),
+  /**
+   * A storage object that can overwrites the default http cache
+   */
+  httpCacheStorage: new ActionContextKey<IHttpCacheStorage>('@communica/actor-http-cache:http-cache-storage'),
```

Comment:
This one can be moved to `KeysHttp`, and 
```suggestion
  httpCacheStorage: new ActionContextKey<IHttpCacheStorage>('@communica/bus-http:cache-storage'),
```